### PR TITLE
[BST-13457] Removes prefix BoostSecurity for 3rd party scanners.

### PR DESCRIPTION
**Pull request by stlef14**
For 3rd party scanners, we're to remove the prefix BoostSecurity, recommended to avoid issues with the 3rd parties.

### CODE REVIEW

File: scanners/boostsecurityio/bundler-audit/module.yaml [hunk 1]
```diff
@@ -2,7 +2,7 @@ api_version: 1.0
 
 
 id: boostsecurityio/bundler-audit
-name: BoostSecurity bundler-audit
+name: Bundler-audit
```
Comment (lindycoder) [inline scanners/boostsecurityio/bundler-audit/module.yaml:5]: Are you sure the case should change, name seems to be in lower case
https://github.com/rubysec/bundler-audit

File: scanners/boostsecurityio/gosec/module.yaml [hunk 2]
```diff
@@ -2,7 +2,7 @@ api_version: 1.0
 
 
 id: boostsecurityio/gosec
-name: BoostSecurity gosec
+name: Gosec
```
Comment (lindycoder) [inline scanners/boostsecurityio/gosec/module.yaml:5]: Here too it seems lower case
https://github.com/securego/gosec

File: scanners/boostsecurityio/npm-audit/module.yaml [hunk 3]
```diff
@@ -2,7 +2,7 @@ api_version: 1.0
 
 
 id: boostsecurityio/npm-audit
-name: BoostSecurity npm-audit
+name: Npm-audit
```
Comment (lindycoder) [inline scanners/boostsecurityio/npm-audit/module.yaml:5]: Seems linke a tool from `npm` which is lower case, should it change?
https://docs.npmjs.com/cli/v9/commands/npm-audit

File: scanners/boostsecurityio/osv-scanner/module.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ api_version: 1.0
 
 
 id: boostsecurityio/osv-scanner
-name: BoostSecurity osv-scanner
+name: OSV Scanner
```
Comment (lindycoder) [inline scanners/boostsecurityio/osv-scanner/module.yaml:5]: According to https://github.com/google/osv-scanner maybe it should be OSV-Scanner  with a dash

File: scanners/boostsecurityio/semgrep/module.yaml [hunk 5]
```diff
@@ -1,7 +1,7 @@
 api_version: 1.0
 
 id: boostsecurityio/semgrep
-name: BoostSecurity semgrep
+name: Ssemgrep
```
Comment (lindycoder) [inline scanners/boostsecurityio/semgrep/module.yaml:4]: (IMPORTANT) There's an extra s here, `Semgrep` seems to be the right casing

**Review by lindycoder — COMMENTED** [review]
Seems some casing are not exactly what the product advertise, I don't know if we care, you can ignore if we dont.

but `Ssemgrep` has a typo

File: scanners/boostsecurityio/gosec/module.yaml (hunk 2 above)
Comment (stlef14) [reply scanners/boostsecurityio/gosec/module.yaml]: fixed

File: scanners/boostsecurityio/bundler-audit/module.yaml (hunk 1 above)
Comment (stlef14) [reply scanners/boostsecurityio/bundler-audit/module.yaml]: fixed

File: scanners/boostsecurityio/npm-audit/module.yaml (hunk 3 above)
Comment (stlef14) [reply scanners/boostsecurityio/npm-audit/module.yaml]: fixed

File: scanners/boostsecurityio/semgrep/module.yaml (hunk 5 above)
Comment (stlef14) [reply scanners/boostsecurityio/semgrep/module.yaml]: fixed

**Review by lindycoder — APPROVED** [review]

**Pull request merged**